en/guide/directory-structure.md

```diff
@@ -9,7 +9,7 @@ description: The default Nuxt.js application structure is intended to provide a
 
 ### The Assets Directory
 
-The `assets` directory contains your un-compiled assets such as LESS, SASS or JavaScript.
+The `assets` directory contains your un-compiled assets such as Less, Sass or JavaScript.
```

Comment:
« Less » and « Sass » are written like this as in official documentations
— https://github.com/nuxt/docs/pull/230/files#r138933898